subprojects/docs/src/docs/userguide/buildCache.adoc

```diff
@@ -248,6 +248,10 @@ These issues may affect the number of cache hits you get between machines.
 | If two or more tasks share an output directory or files, Gradle will disable caching for these tasks when it detects an overlap.
 | Use separate output directories for each task.
 
+| Using cached C/C++ object files with absolute paths
+| When Gradle compiles C/C++ code, object files tend to have absolute paths embedded inside them. This doesn't affect their correctness, but it can interfere with debuggers that search for source code at those absolute paths.
+| Use identical absolute paths for projects sharing a build cache.
```

Comment:
I might make it a little clearer what it means for projects to be sharing a build cache. Maybe, "projects sharing a build cache across machines." or something like that?

Reply:
I'll try to clarify it.  

It can also mean the same project checked out in different paths on the same machine.